test/e2e/authorizer/authorizer_test.go

```diff
@@ -70,10 +69,10 @@ func TestAuthorizer(t *testing.T) {
 	createResources(t, ctx, dynamicClusterClient, kubeClusterClient.DiscoveryClient, org1, "org-resources.yaml")
```

Comment:
nit: we might want to create the workspaces with the framework util first, and not in this, as we will likely start adding utilities layered on top of `framework.NewWorkspaceFixture()` that will be harder to make function if we are not assuming that we're the only one creating them

Reply:
this seems to be unrelated to this PR, do you mind to submit an issue?

Reply:
You already did it in this PR :P